src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +344,40 @@ Demos
 
 ## FAQ
 
+**Do I need the `--disable-gpu` flag?**
+
+Yes, for now.  The `--disable-gpu` flag is a temporary requirement to work around a few bugs. You won't need this
+flag in future versions of Chrome. See [https://crbug.com/546953#c152](https://bugs.chromium.org/p/chromium/issues/detail?id=546953#c152) and [https://crbug.com/695212](https://bugs.chromium.org/p/chromium/issues/detail?id=695212) for more information.
+
+**So I still need Xvfb?**
```

Comment:
On second look, looks like you could create a section around automated testing, and put each of these FAQs as subsections. That would make it more skimmable (which, I think, is the main reason why writers see FAQs as a code smell)

I don't care much if you do that or not, I'm just trying to be more helpful as a writer and provide rationale for why we suggest the things we do, rather than giving vague high-level remarks that seem arbitrary